enhancements/management-workload-partitioning.md

```diff
@@ -353,14 +353,14 @@ scheduled to run on the management CPU pool.
 7. The kubelet reads static pod definitions. It replaces the `cpu`
    requests with `management.workload.openshift.io/cores` requests of
    the same value and adds the
-   `io.openshift.workload.management.cpushares/{container-name}`
+   `resources.workload.openshift.io/{container-name}: {"cpushares": N}`
```

Comment:
I am unsure regarding `N`, should it be the integer or the string `"N"`?

Reply:
I think it better to make it string, in case if we will support additional parameters in the future(for example cpuset that will have value 0-3 and can be the integer)

Reply:
Sorry, that should be an integer value, I just didn't have a specific number to put there. I'll expand the example to include a number.

Reply:
@dhellmann  I saw the example below, so it should be ok.
Do you know if will we always have only CPU shares or does it possible we will have an additional resource in the future(CPUSet for example). Something like `{"cpushares": 256, "cpuset": "0-3"}`.

Reply:
I updated these examples to use real numbers for clarity.